script.js

```diff
@@ -0,0 +1,9 @@
+const fs = require('fs')
+const filename = process.argv[2];
+const name = process.argv[3]
+let greeting = `console.log(Hello ${name})`
+
+fs.writeFile(filename, greeting, (err) => {
+  if (err) throw err;
+  console.log(`Data has been written`)
```

Comment:
Nitpick but avoid template literals for strings that don't need interpolation. It will make your code easier to scan when you can see which strings are dynamic.